src/components/UsersList/UsersList.tsx

```diff
@@ -112,13 +122,89 @@ export function UsersList({
     openModal(user);
   }
 
+  // Overlays
+  const openOverlay = (userId: string, type: OverlayType = 'org') => {
+    setOverlayType(type);
+    setCurrentMemberId(userId);
+    setOverlayOpen(true);
+  };
+
+  const closeOverlay = () => {
```

Comment:
Hmm I seem to be getting inconsistent behavior with this one (Safari only). A few hours ago, I saw this happen as well, but now it's returning to the button as expected. I couldn't find any literature suggesting that Safari should behave differently here, but if I had to guess, maybe the `tabindex="0"` on the table container (needed for the scrollable table) might be throwing it off.

Reply:
@cannandev @hursey013 Let me know if you see this happening as well

Reply:
I think I might’ve figured out how to reproduce this fairly reliably, at least over here:

1. If I open an overlay via the keyboard, focus is returned correctly to the opening element.
2. If I open an overlay with a mouse/pointer, focus returns to the table.

Are you seeing that on your end?

Reply:
Interesting—I'm seeing it now. I think my `tabindex="0"` suspicion is correct: when I remove that from the table container and open with a click, focus just returns to the very top of the page when closing out. (Ugh)

My guess is that only sighted users will be using the mouse to click on buttons, so this may not be a huge issue for them. For those that click buttons through the keyboard, focus will be returned as expected. But this difference in behavior between browsers is indeed frustrating.

Reply:
Woof, well, glad you’re also able to reproduce it!

I’d love to hear what you/Claire/Brian think, but I’d be inclined to think of this as a decently-sized a11y bug? Some people may use a screen reader and a mouse, possibly in conjunction with some screen magnification software. ([VoiceOver](https://support.apple.com/guide/voiceover/use-cursor-tracking-vo15534/mac) and [NVDA](https://www.nvaccess.org/files/nvda/documentation/userGuide.html#NavigatingWithTheMouse) both support cursor tracking.) I don’t think this needs to block this PR, necessarily, but I might suggest logging this as a UX issue.

Reply:
All good points, and I'm very willing to defer to others here.

Reply:
I agree with @beepdotgov. The issue doesn't need to block this PR, but let's log as an important UX issue.

Reply:
Logged in #496!